ui/accordion.info/ui/accordion.reel/accordion.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * @module accordion
+ * @requires montage/ui/montage
+ * @requires montage/ui/component
+ */
+var Montage = require("montage").Montage,
+    Component = require("montage/ui/component").Component;
+
+/**
+ * @class Accordion
+ * @extends Component
+ */
+exports.Accordion = Montage.create(Component, {
+
+    /**
+     * @function prepareForActivationEvents
+     * @param null
+     * @description Add the the event listener for this class object
+     * @private
+     */
+    prepareForActivationEvents: {
+        value: function () {
+            if (window.Touch) {
```

Comment:
A press Composer needs to be used here, but with a repetition doing the work of repeating the array of components passed, the repetition handles the selection, so we might be able to get what we want from there